src/Sidebar/Sidebar.tsx

```diff
@@ -8,12 +7,43 @@ import {
   SidebarHeaderStyles,
 } from "./sidebar.style";
 
-const renderableChildrenPropType = PropTypes.oneOfType([
-  PropTypes.arrayOf(PropTypes.node),
-  PropTypes.node,
-]);
+type LinkProps = {
+  iconName: string;
+  active?: boolean;
+  href: string;
+  children?: ReactNode;
```

Comment:
not needed. please always remove from types unless needed

Reply:
hahhah, just now removed it before you comment and will push this one looool